radicle-httpd/src/api/v1/sessions.rs

```diff
@@ -41,16 +44,38 @@ async fn session_create_handler(State(ctx): State<Context>) -> impl IntoResponse
     let expiration_time = OffsetDateTime::now_utc()
         .checked_add(UNAUTHORIZED_SESSIONS_EXPIRATION)
         .unwrap();
-    let auth_state = AuthState::Unauthorized {
+    let session = Session {
+        status: AuthState::Unauthorized,
         public_key: *signer.public_key(),
+        issued_at: DateTime(OffsetDateTime::now_utc()),
         expires_at: DateTime(expiration_time),
     };
     let mut sessions = ctx.sessions.write().await;
-    sessions.insert(session_id.clone(), auth_state);
+    sessions.insert(session_id.clone(), session.clone());
+
+    Ok::<_, Error>(Json(json!({
+        "sessionId": session_id,
+        "publicKey": session.public_key,
+        "issuedAt": session.issued_at,
+        "expiresAt": session.expires_at
+    })))
```

Comment:
I'm wondering should all `json!` be placed [here](https://github.com/radicle-dev/heartwood/blob/master/radicle-httpd/src/api/json.rs)? Would help with the duplication.

Reply:
hmm not a bad idea, not sure it would help with the duplication, since we return similar but not equal objects in the GET and POST route.

Reply:
Ok did make both json objects equal and extracted it to `api::json`
741252e